app/views/apis/show.html.erb

```diff
@@ -1,9 +1,12 @@
 <h2><%= @api.name %></h2>
 
 <p>
+  <table>
```

Comment:
Why a table? Also...why a table without any rows or cells? 

Foundation grid divs are more flexible and responsive. Only use tables in HTML when required for semantic meaning, i.e. the tabulated results of something, etc. A list of some kind would be more appropriate here, styled to your liking. Or a set of divs with other divs inside of them to control the discrete parts of each listing.